crates/sui-bridge/src/types.rs

```diff
@@ -492,7 +492,7 @@ impl BridgeAction {
         BridgeActionDigest::new(hasher.finalize().into())
     }
 
-    pub fn source_chain_id(&self) -> BridgeChainId {
+    pub fn chain_id(&self) -> BridgeChainId {
```

Comment:
Why is this better though? source_chain sounds accurate?

Reply:
source_chain is only relevant to token transfer

Reply:
Something like token_transfser_source_chain would be accurate, but I understand it's getting long. Once you introduce a secondary enum this will become easier I guess.